packages/ddp-server/stream_server_tests.js

```
@@ -0,0 +1,66 @@
+// Tests for the _redirectWebsocketEndpoint URL rewriting logic in stream_server.js.
+// Verifies that /websocket requests are correctly redirected to /sockjs/websocket.
+
+var http = Npm.require('http');
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)